pkg/controller/admin/caches.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package admin
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/controller"
+	"github.com/gorilla/mux"
+)
+
+type cacheItem struct {
+	Name        string
+	Description string
+}
+
+var caches = map[string]*cacheItem{
+	"authorized_apps:": {"API keys", "Authentication caches for mobile devices"},
+	"apps:":            {"Mobile apps", "Registered mobile apps for the redirector service"},
+	"jwks:":            {"JWKs", "JSON web tokens"},
+	"realms:":          {"Realms", "Caches of realm data"},
```

Comment:
probably should remove the word cache from all of these descriptions.

Reply:
Done.